docs/recipes/mocking-api-endpoints.md

```diff
@@ -0,0 +1,23 @@
+# Mocking API endpoints
+
+Since _miyagi_ is a lot about developing components independently of a backend framework, we might also want to mock API endpoints, for example to test form submissions using AJAX requests.
```

Comment:
```suggestion
Since _miyagi_ is focused on developing components independently of a backend framework, we might also want to mock API endpoints, for example, to test form submissions using AJAX requests.
```